### PR TITLE
Create trivy.yml

### PR DESCRIPTION
**Pull request by Chris-Sigopt**

### CODE REVIEW

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        uses: actions/checkout@v4
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t docker.io/my-organization/my-app:${{ github.sha }} .
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@7b7aa264d83dc58691451798b4d117d53d21edfe
+        with:
+          image-ref: "docker.io/my-organization/my-app:${{ github.sha }}"
```
Comment (malkomes) [inline .github/workflows/trivy.yml:39]: `my-organization` and `my-app` will be placeholders or do we want to have it as a variable?

**Review by malkomes — APPROVED** [review]
lgtm

File: .github/workflows/trivy.yml (hunk 1 above)
Comment (Chris-Sigopt) [reply .github/workflows/trivy.yml]: Yeah, I'm hoping that was the issue that I didn't realize they were placeholders.

**Comment by Chris-Sigopt** [comment]
Already had this under the security github action.

**Pull request closed without merging**